modules/ROOT/pages/_partials/api-governance.adoc

```diff
@@ -709,7 +746,8 @@ anypoint-cli-v4 governance:ruleset:validate ~/temp/myrulesetfolder
 *Example output for a valid ruleset:*
 
 ----
- Ruleset conforms with Dialect
+Ruleset User/myuser/temp/myruleset.yaml
```

Comment:
Is this also supposed to be `myrulesetfolder` instead of 'temp'?

Reply:
No, I just changed a couple so people knew it didn't have to be temp. Thanks for asking.